src/themes/Svingle/Footer/SmallPrintFooter.scss

```diff
@@ -0,0 +1,61 @@
+@use 'src/components/Footer/SmallPrintFooter';
+@use '../variables/breakpoints';
+
+$right-column-width: 300px !default;
+$left-column-margin: 0 30px 0 0 !default;
+
+.wmde-banner {
+	@include SmallPrintFooter.layout;
+
+	&-footer {
+		font-size: 12px;
+		line-height: 20px;
+		color: var( --footer-color );
+		padding: 0;
+		margin-top: 0;
+
+		&-bank {
+			padding: 5px 10px 5px 0;
+			text-align: right;
+			&-item {
+				padding-left: 0.5em;
+
+				&.account .selection-input-text,
+				&.account .selection-input-input {
+					text-transform: none;
+					white-space: nowrap;
+				}
+			}
+		}
+
+		&-usage {
+			color: #72777d;
+			padding: 5px 0;
+			margin: $left-column-margin;
+			text-align: left;
+			&-link,
+			&-link:visited {
+				color: #72777d;
```

Comment:
This is a good chance to have a play with the swatches. You can add a `--footer-smallprint-color` variable into the light and dark colours and use it here with `var( --footer-smallprint-color )`.

You can optionally add a new flag to the mixin and the skin file to optionally include the variable, but that's up to you. I reckon we'll be using this footer for the campaign.